frontend/apps/reader/modules/readergoto.lua

```diff
@@ -119,4 +119,17 @@ function ReaderGoto:gotoPage()
     end
 end
 
+function ReaderGoto:gotoBeginning()
```

Comment:
Btw, for calling purposes it's easier to use an event that can be called from the gesture manager.